agent/outbound.go

```diff
@@ -21,23 +21,23 @@ import (
 )
 
 // listAppCallback
-func (a *Agent) listAppCallback(apps []v1alpha1.Application) []v1alpha1.Application {
-	logCtx := log().WithField("event", "ListApps")
-	logCtx.Debugf("List apps event")
-
-	// Every time we're relisting, we are clearing the managed apps list
-	// and re-add all apps returned by the lister.
-	a.appManager.ClearManaged()
-	for _, app := range apps {
-		if err := a.appManager.Manage(app.QualifiedName()); err != nil {
-			log().Warnf("Could not manage app %s: %v", app.QualifiedName(), err)
-		}
-		if err := a.appManager.IgnoreChange(app.QualifiedName(), app.ResourceVersion); err != nil {
-			log().Warnf("Could not ignore change %s for app %s: %v", app.ResourceVersion, app.QualifiedName(), err)
-		}
-	}
-	return apps
-}
+// func (a *Agent) listAppCallback(apps []v1alpha1.Application) []v1alpha1.Application {
```

Comment:
Do we plan to use this function again? or should we remove it?

Reply:
Oops. This should be gone indeed :)